DAQ_System/Source/main.cpp

```diff
@@ -40,6 +40,15 @@
 #define BUFFER_SIZE 21 // (4 linpot integers) * (4 bytes allocated per integer) = 16, 4 commas = 4 chars = 4 bytes, 1 more char & byte for the newline, 16 + 4 + 1 = 21
 using namespace application;
 
+InterruptIn pb(PC_13);
+
+uint8_t data_logger_flag= 0;
```

Comment:
- Rename the button interrupt input object
- Does this object have to be in the global scope? If so, make it static
- Make `data_logger_flag` static
- Perhaps to rename to something like `logging_enabled`?? You can probably be more creative than me lol